azure/create-platform-prerequisites/azure.tf

```diff
@@ -18,6 +20,7 @@ provider "azurerm" {
   client_id       = var.client_id
   client_secret   = var.client_secret
   tenant_id       = var.tenant_id
+  skip_provider_registration = true
```

Comment:
what is the purpose of this parameter ?

Reply:
It is to ensure that azure will not register it own providers 
<https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_provider_registration>

Reply:
I understand that but are we sure we want to handle on our side the provider resources we need? I mean, why not letting azurerm provider handle this ?